getting-started.md

```diff
@@ -1,35 +1,44 @@
 # Getting Started
 
-## Creating Account
+### Creating Account
 
-You will need the Exogress account to use the service. Please, visit [https://app.exogress.com](https://app.exogress.com/) and create a new account.
+Visit [https://app.exogress.com](https://app.exogress.com/) to create a new account and follow the instructions. Make sure you have activated your account.
```

Comment:
activation is required only if user registered through email. In case of google or github activation is not required